.gitignore

```diff
@@ -19,3 +19,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```

Comment:
I think we would need this kind of file in the future, I recommend to commit it to github